packages/server-boilerplate/src/orchestrator/session/attachSession.ts

```diff
@@ -11,7 +11,7 @@ export const attachSession = async (req: Request, res: Response, next: NextFunct
   try {
     const sessionId = req.sessionCookie?.id;
     if (!sessionId) {
-      throw new UnauthenticatedError('User not authenticated');
+      throw new UnauthenticatedError('Session not found or has expired. Please log in again.');
```

Comment:
Tom suggested a nice idea to which is to use a new http response code to indicate that the session has expired (https://github.com/beyondessential/tupaia-backlog/issues/3233#issuecomment-887094229). However, I think with the case of session cookie not found, it could be because of some other reasons rather than just the session has expired (eg: someone unauthenticated trying to access the server). So I'm still keeping 401 as the response code here.